twitchio/user.py

```diff
@@ -621,7 +622,7 @@ def __init__(self, http: "TwitchHTTP", data: dict):
         self.title: str = data["title"]
         self.thumbnail_url: str = data["thumbnail_url"]
         self.live: bool = data["is_live"]
-        self.started_at = datetime.datetime.strptime(data["expires_at"], "%Y-%m-%dT%H:%M:%SZ") if self.live else None
+        self.started_at = datetime.datetime.strptime(data["started_at"], "%Y-%m-%dT%H:%M:%SZ") if self.live else None
         self.tag_ids: List[str] = data["tag_ids"]
```

Comment:
Ideally this would best be done in its own pull request, as it's not relevant to this one

Reply:
I've removed this change. Will follow up with another PR.